src/lightning_app/cli/lightning_cli_create.py

```diff
@@ -33,14 +33,14 @@ def create():
     help="Instance types that you want to support, for computer jobs within the cluster.",
 )
 @click.option(
-    "--cost-savings",
-    "cost_savings",
+    "--enable-performance",
```

Comment:
@nicolai86 shouldn't this be added to the changelog?

Reply:
I'm torn - on the one hand yes, because it's a breaking change to the CLI.
On the other hand no, because nobody can use the CLI today. 

Let me follow the proper process and add a changelog entry here.